scripts/build/process-args.sh

```diff
@@ -4,25 +4,31 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
 
-for i in "$@"
+params=("$@")
+
+for i in "${!params[@]}"
 do
-    lowerI="$(echo $i | awk '{print tolower($0)}')"
+    lowerI="$(echo ${params[$i]} | awk '{print tolower($0)}')"
     case $lowerI in
-    release)
+    "release" | "-release")
```

Comment:
Yes. I initially wanted to add short name support ('-r' for 'release') and decided against it. Let me fix it